src/docs/product/performance/metrics.mdx

```diff
@@ -81,6 +81,12 @@ User Misery is a user-weighted performance metric to assess the relative magnitu
 
 You can set satisfactory thresholds for each project with [custom thresholds](#custom-thresholds).
 
+<Note>
+
+User Misery is calculated using a probability distribution. This prevents false positives and ensures reasonable values for data sets with low event volume.
```

Comment:
Meant to add my comment to this line!

Reply:
I figure this is fine! I could have said "Beta Distribution" but I'm choosing to wave my hands a bit and say "it's statistical" so people get the idea that it's non-trivial. I think anyone who's using Sentry has at least _heard_ of statistics, right?

Reply:
I'd maybe clarify a bit, since `User Misery`, the rate / visualization are a prob. distribution, but the underlying data `# of miserable users` should always be correct.

Reply:
@k-fish clarified in https://github.com/getsentry/sentry-docs/pull/5402/commits/26bd98b99c2f30e350ce6e0a16654106aea9833d. How's that?

Reply:
It should help 👍